lib/utils/notifications.dart

```diff
@@ -1,129 +0,0 @@
-import 'package:flutter/material.dart';

```

Comment:
Most of this logic was also moved to `lib/notification/utils/local_notifications.dart` and `lib/notification/shared/android_notification.dart`